content/docs/troubleshooting/_index.md

```diff
@@ -473,3 +473,35 @@ value defined elsewhere in your application. If the value is known, it can be co
 
 If the stack-reference-name truly is dynamic and cannot be known ahead of time to supply directly into the app, then this
 approach will not work, and the only way to workaround the issue is to follow the steps in [Use getOutput/requireOutput](#use-getoutput).
+
+## 403 HTTP error fetching plugin
+
+You're more than likely seeing this error message as you're using an arm64 based processor (probably one of the new M1 MacBook Pros) and are using an older version of one of our providers that doesn't support this processor.
+
+It's not possible just to upgrade the packages as your state will still be locked to the old version of the provider.
+
+There are two ways to fix this, one way if you have access to an Intel based computer and one if you don't
+
+### I have access to an Intel based computer
+
+1. Open your Pulumi program on a non-arm64 based computer
+1. Update your packages (pip / nuget / npm / go) and run `pulumi up`
+1. Once the update is complete, you can open the new, updated Pulumi program on your arm64-based system
+
+### I don't have access to an Intel based computer
+
+1. Remove Pulumi - if you're using Homebrew, `brew remove pulumi` or simply `rm -rf ~/.pulumi`
+1. Download latest version of Pulumi: `https://get.pulumi.com/releases/sdk/pulumi-v{version}-darwin-x64.tar.gz` (current version is [https://get.pulumi.com/releases/sdk/pulumi-v2.24.0-darwin-x64.tar.gz](https://get.pulumi.com/releases/sdk/pulumi-v2.24.0-darwin-x64.tar.gz)) and extract to ~/.pulumi/bin
+1. Add Pulumi to path: `export PATH=$PATH:~/.pulumi/bin`
+1. Update packages in your Pulumi program to latest version (for example `npm install @pulumi/aws@latest)
+1. Install Pulumi provider: `arch -x86_64 pulumi plugin install resource {provider_name} v{version}` (where  {provider_name} is the name of the provider, i.e. aws and {version} is the same version number that your package has updated to)
+1. Login to Pulumi using the appropriate backend (see [our docs](https://www.pulumi.com/docs/intro/concepts/state/#logging-in) for information)
```

Comment:
Is the `login` step needed? Presumably if there in this state, they're likely already logged in to their backend or familiar with how to do that if they get an error about it.

Reply:
We're deleting the `.pulumi` folder so they'll need to login to get the credentials

Reply:
Ah, given that, I think we should adjust these instructions. Deleting `.pulumi` could delete state files, backups, etc. that we likely do not want to delete. Can we delete `.pulumi/bin` and `.pulumi/plugins` instead of the main folder?